crates/router/src/core/payment_link/payment_link_initiate/payment_link.js

```diff
@@ -417,11 +417,10 @@ function initializeSDK() {
       ? "accordion"
       : paymentDetails.sdk_layout;
 
-  var enabledSavedPaymentMethod = paymentDetails.enabledSavedPaymentMethod;
-
+  var enableSavedPaymentMethod = paymentDetails.enabled_saved_payment_method;
   var unifiedCheckoutOptions = {
-    displaySavedPaymentMethodsCheckbox: enabledSavedPaymentMethod,
-    displaySavedPaymentMethods: enabledSavedPaymentMethod,
+    displaySavedPaymentMethodsCheckbox: enableSavedPaymentMethod,
```

Comment:
Do we have use cases where we need to display checkbox but not SPMs?

Reply:
Seperate fields are need by sdk to invoke saved payment method flow for customer